postgres/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 
 ### Changes
 
+* [MINOR] Allow specifying postgres port as string.
```

Comment:
Can I ask a small favor? Could you add a new entry in the changelog?

`1.0.4 / Unreleased` - and add this fix there?